### ISSUE TITLE
Aint working with TableViewController 

### ISSUE THREAD

**Issue by Aviad94** [kciter/Floaty#286]
is there any solution for that? :(

**Comment by Aviad94** [comment]
Lol found the problem, just changed it to - self.navigationController?.view.addSubview(floaty) and it works xD hope i helped people ...

**Comment by a-choyster** [comment]
Yup, your solution works and it's useful if you want floaty to appear in all of the ViewControllers in your NavController stack. But if you want floaty for just a single TableViewController, then you're better off using:

floaty.sticky = true
self.tableView.addSubview(floaty)

**Comment by fukemy** [comment]
@a-choyster i have tabbar -> tableview, but the fab is under tabbar

**Comment by BrettJackson1987** [comment]
> @a-choyster i have tabbar -> tableview, but the fab is under tabbar

Just add the height of the tab bar to the y-axis padding:

floaty.paddingY += tabBarController!.tabBar.bounds.size.height
